custom/succeed/models.py

```diff
@@ -107,4 +107,43 @@ class UCLAPatientFluff(fluff.IndicatorDocument):
 
     emitter = RandomizationDate()
 
+
+def get_full_name(case):
+    try:
+        return CommCareCase.get(case['indices'][0]['referenced_id'])['full_name']
+    except AttributeError:
+        return None
+
+
+class Emitter(fluff.Calculator):
+
+    @fluff.null_emitter
+    def date(self, case):
+        yield None
+
+
+class UCLATaskActivityFluff(fluff.IndicatorDocument):
```

Comment:
@czue @snopoke 
Added Simon because If I good remember he mostly works on this.

I have question regarding CustomDataSoutce. Is there are possibility to get property from list of dict's?
example:
```
    ... ,
    "property": [
         { "some_property": 1 ...}
         ...
    ]
```

I want get ```some_property``` this is always in first dict in list.
I tried something like this:
```
 {
         "column_id": "column_id",
         "type": "expression",
         "expression": {
              "type": "property_path",
                    "property_path": ["property", "0", "some_property"]
               },
         "datatype": "string"
},
```
but this not work